src/app/api/transform/conference.0/route-conference-archived.ts

```diff
@@ -3,6 +3,7 @@ import { eq } from "drizzle-orm"
 import { ConferenceEventArchivedPayload } from "@/contracts/events/conference"
 import { db } from "@/database"
 import { conferences } from "@/database/schemas"
+import { archiveProduct } from "@/lib/stripe/product"
```

Comment:
Just a quick question, if this ought to reside in the lib folder on the server side?
Or are they inaccessible regardless after the build, through NextJS magic? :)

Reply:
They should be inaccessible. I don't think the new nextjs uses folder structure to decide what it exposed and what is not.